gamestonk_terminal/cryptocurrency/defi/defi_controller.py

```diff
@@ -622,3 +629,193 @@ def call_swaps(self, other_args: List[str]):
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="vaults",
+            description="""
+                Display Top DeFi Vaults.
+                [Source: https://coindix.com/]
+            """,
+        )
+
+        parser.add_argument(
+            "-c",
+            "--chain",
+            dest="chain",
+            type=str,
+            help="Blockchain name e.g. ethereum, terra",
+            default=None,
+            choices=coindix_model.CHAINS,
+            required=False,
+        )
+
+        parser.add_argument(
+            "-p",
+            "--protocol",
+            dest="protocol",
+            type=str,
+            help="DeFi protocol name e.g. aave, uniswap",
+            default=None,
+            choices=coindix_model.PROTOCOLS,
+            required=False,
+        )
+
+        parser.add_argument(
+            "-k",
+            "--kind",
+            dest="kind",
+            type=str,
+            help="Kind/type of vault e.g. lp, single, noimploss, stable",
+            default=None,
+            choices=coindix_model.VAULT_KINDS,
+            required=False,
+        )
+
+        parser.add_argument(
+            "-l",
+            "--limit",
+            dest="limit",
+            type=check_positive,
+            help="Number of records to display",
+            default=10,
+        )
+
+        parser.add_argument(
+            "-s",
+            "--sort",
+            dest="sortby",
+            type=str,
+            help="Sort by given column. Default: timestamp",
+            default="apy",
+            choices=coindix_model.VAULTS_FILTERS,
+        )
+
+        parser.add_argument(
+            "--descend",
+            action="store_false",
+            help="Flag to sort in descending order (lowest first)",
+            dest="descend",
+            default=False,
+        )
+
+        ns_parser = parse_known_args_and_warn(
+            parser, other_args, EXPORT_ONLY_RAW_DATA_ALLOWED
+        )
+
+        if ns_parser:
+            coindix_view.display_defi_vaults(
+                chain=ns_parser.chain,
+                kind=ns_parser.kind,
+                protocol=ns_parser.protocol,
+                top=ns_parser.limit,
+                sortby=ns_parser.sortby,
+                descend=ns_parser.descend,
+                export=ns_parser.export,
+            )
+
+    def print_help(self):
+        """Print help"""
+        help_text = """
+Decentralized Finance Menu:
+
+Overview:
+    llama         DeFi protocols listed on DeFi Llama
+    tvl           Total value locked of DeFi protocols
+    newsletter    Recent DeFi related newsletters
+    dpi           DeFi protocols listed on DefiPulse
+    funding       Funding rates - current or last 30 days average
+    borrow        DeFi borrow rates - current or last 30 days average
+    lending       DeFi ending rates - current or last 30 days average
+    vaults        Top DeFi Vaults on different blockchains
```

Comment:
Missing [Source: Coindix]

Reply:
Should we add [Source: Coindix] at print() method level e.g.
```
vaults        Top DeFi Vaults on different blockchains [Source: Coindix]
```
?

Reply:
yea i think since it is the only command this is it